### PR TITLE
Fixing conf path to be taken from system property

### PR DESCRIPTION
**Pull request by himasha**

### CODE REVIEW

File: components/andes/org.wso2.carbon.andes/src/main/java/org/wso2/carbon/andes/service/QpidServiceImpl.java [hunk 1]
```diff
@@ -59,7 +61,9 @@
     /**
      * The location for the qpid configuration file directory.
      */
-    private static final String QPID_CONF_DIR = "/repository/conf/advanced/";
+    public static final String QPID_CONF = "qpid.conf";
+    private static String qpidPath;
```
Comment (hastef88) [inline components/andes/org.wso2.carbon.andes/src/main/java/org/wso2/carbon/andes/service/QpidServiceImpl.java]: If we are only using qpidPath variable in one method, might as well make it a method level variable.

File: components/andes/org.wso2.carbon.andes.ui/src/main/java/org/wso2/carbon/andes/ui/UIUtils.java [hunk 2]
```diff
@@ -47,13 +48,16 @@
...
 
 /**
  * This class is used by the UI to connect to services and provides utilities. Used by JSP pages.
  */
 public class UIUtils {
 
-    private static final String ANDES_CONF_DIR = "/repository/conf/advanced/";
+    public static final String QPID_CONF = "qpid.conf";
+    private static String qpidPath = System.getProperty(QPID_CONF);
+    private static  String ANDES_CONF_DIR = "/repository/conf/advanced/";
```
Comment (akalankapagoda) [inline components/andes/org.wso2.carbon.andes.ui/src/main/java/org/wso2/carbon/andes/ui/UIUtils.java]: The value of ANDES_CONF_DIR changes. So IMO this should be a variable with camel case.

File: components/andes/org.wso2.carbon.andes.ui/src/main/java/org/wso2/carbon/andes/ui/UIUtils.java [hunk 3]
```diff
@@ -273,6 +277,9 @@ public static String getTCPConnectionURL(String userName, String accessKey)
 
         // these are the properties which needs to be passed when ssl is enabled
         String CARBON_SSL_PORT = String.valueOf(AndesConfigurationManager.readValue(AndesConfiguration.TRANSPORTS_AMQP_SSL_CONNECTION_PORT));
+        if(qpidPath != null){
```
Comment (akalankapagoda) [inline components/andes/org.wso2.carbon.andes.ui/src/main/java/org/wso2/carbon/andes/ui/UIUtils.java]: Needs to format this.

File: components/andes/org.wso2.carbon.andes/src/main/java/org/wso2/carbon/andes/service/QpidServiceImpl.java [hunk 4]
```diff
@@ -296,6 +300,12 @@ public String getInternalTCPConnectionURL(String username, String password, Stri
      */
     @Override
     public String getQpidHome() {
+        //get system property value for qpid-config.xml if exists
+        qpidPath = System.getProperty(QPID_CONF);
+
+        if(qpidPath != null){
```
Comment (akalankapagoda) [inline components/andes/org.wso2.carbon.andes/src/main/java/org/wso2/carbon/andes/service/QpidServiceImpl.java]: Needs to format.

File: components/andes/org.wso2.carbon.andes/src/main/java/org/wso2/carbon/andes/service/QpidServiceImpl.java (hunk 1 above)
Comment (himasha) [inline components/andes/org.wso2.carbon.andes/src/main/java/org/wso2/carbon/andes/service/QpidServiceImpl.java]: Fixed with commit 	a42bcc9

File: components/andes/org.wso2.carbon.andes/src/main/java/org/wso2/carbon/andes/service/QpidServiceImpl.java (hunk 4 above)
Comment (himasha) [inline components/andes/org.wso2.carbon.andes/src/main/java/org/wso2/carbon/andes/service/QpidServiceImpl.java]: Fixed with commit a42bcc9

File: components/andes/org.wso2.carbon.andes.ui/src/main/java/org/wso2/carbon/andes/ui/UIUtils.java (hunk 3 above)
Comment (himasha) [inline components/andes/org.wso2.carbon.andes.ui/src/main/java/org/wso2/carbon/andes/ui/UIUtils.java]: Fixed with commit  a42bcc9

**Pull request merged**